### ISSUE TITLE
Plasma lockscreen and ctrl+alt+del screen have gaps the size of the dock on the bottom

### ISSUE THREAD

**Issue by elkcl** [anametologin/krohnkite#98]
When using the default krohnkite settings, the lockscreen and the ctrl+alt+del screen have a gap at the bottom where the dock usually resides (unfortunately, taking a screenshot of either to demonstrate is kind of problematic). This is fixed by adding `kwin_wayland` and `ksmserver-logout-greeter` to the ignored window classes respectfully.

Maybe adding them to the default settings is a good idea?

**Environment**
 - Distro: Archlinux
 - KWin version: 6.2.2
 - Krohnkite version: fd67a3acd0a220bb6c1f41a1c7f3cc538ef19a82
 - List of KWin scripts in use: krohnkite only

**Comment by anametologin** [comment]
If remove `kwin_wayland`, will the problem come back?

**Comment by elkcl** [comment]
Yes, partially. Adding `ksmserver-logout-greeter` to the ignorelist only resolves the problem with the ctrl+alt+del menum and adding `kwin_wayland` only fixes the lockscreen.
